hooks/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+eval $(cat versions.mk)
+
+echo "Building image with versions..."
+echo "NODE_VERSION ${NODE_VERSION}"
+echo "NPM_VERSION ${NPM_VERSION}"
+echo "V8_VERSION ${V8_VERSION}"
+
+docker build \
+	--build-arg NODE_VERSION=${NODE_VERSION} \
+	--build-arg NPM_VERSION=${NPM_VERSION} \
+	--build-arg V8_VERSION=${V8_VERSION} \
+	-t ${IMAGE_NAME} .
```

Comment:
Do you have these hooks enabled in Docker Hub? I had disabled them for the CentOS image builds.

Reply:
I've not enabled anything on dockerhub. Can this be deleted?

Reply:
Yes I believe that everything in `hooks` is for building on Docker Hub. Should probably be removed from `centos7-s2i-nodejs ` as well.

Reply:
Yes I believe that everything in `hooks` is for building on Docker Hub. Should probably be removed from `centos7-s2i-nodejs ` as well.